pkg/k8s/watchers/subscriber/ciliumnode.go

```diff
@@ -28,19 +34,49 @@ type CiliumNode interface {
 type CiliumNodeChain struct {
```

Comment:
Should we just rename this to `LocalCiliumNodeChain` so there won't be confusion in case refactoring takes a bit longer?